odoo/models.py

```diff
@@ -2488,11 +2488,16 @@ def _read_group_format_result(self, data, annotated_groupbys, groupby, domain):
                             locale=locale
                         )
                     data[gb['groupby']] = ('%s/%s' % (range_start, range_end), label)
+                    data.setdefault('__range', {})[gb['groupby']] = {'from': range_start, 'to': range_end}
```

Comment:
Thanks for the review!
There is currently no added benefit, it's arbitrary based on my own preference, and on the eventuality we would want to store additional information later. I'll update the PR to use the pair instead!

Reply:
@rco-odoo 
```python
from collections import namedtuple
groupby_range = namedtuple('Range', ('start', 'end'))
```
Would you use this for readability?

Reply:
Sorry, I realized after my review that the existing code was already based on that dictionary.  I don't think it is a good idea to change the range format in this pull request, because it will generate noise that is not in the scope of the task.
So I suggest to reject my proposal above for now :smile: